src/common/JSONImporter.js

```diff
@@ -9,6 +9,47 @@ define([
         META_ASPECT_SET_NAME: 'MetaAspectSet',
     };
 
+    class NodeAttributesFilter {
```

Comment:
Two thoughts around naming:
- I would avoid "attributes" since it is overloaded (ie, node attributes are a very specific and different thing in webgme)
- it's not clear from "filter" if it is an allow-list or deny-list

Reply:
Renamed to `OmittedWJIProperties`.